src/iotjs.c

```diff
@@ -55,11 +55,13 @@ static bool jerry_initialize(iotjs_environment_t* env) {
 
   if (iotjs_environment_config(env)->debugger != NULL) {
     uint16_t port = iotjs_environment_config(env)->debugger->port;
+#ifdef JERRY_DEBUGGER
```

Comment:
If the debugger is disabled why not disable the whole debugger part here? (from line 56 to 65).
Currently some of the builds on Travis are failing due to the unused port variable. (note: as it is used on line 64, but that is only a debug logging which is not present in release builds, thus the port variable will be unused)